### PR TITLE
plex: add enablePlexPass

### PR DESCRIPTION
**Pull request by colemickens**

### CODE REVIEW

File: pkgs/servers/plexpass/default.nix [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+stdenv.lib.overrideDerivation plex (attrs: rec {
+  name = "plexpass-${version}";
+  version = "0.9.15.3.1674";
+  vsnHash = "f46e7e6";
+
+  src = fetchurl {
+    url = "https://downloads.plex.tv/plex-media-server/${version}-${vsnHash}/plexmediaserver-${version}-${vsnHash}.x86_64.rpm";
+    sha256 = "086njnjcmknmbn90mmvf60ls7q73g2m955yk621jjdngs4ybvm19";
+  };
```
Comment (jagajaga) [inline pkgs/servers/plexpass/default.nix]: Why no meta information?

**Comment by jagajaga** [comment]
Please also name commit as it's written in `CONTRIBUTING.md`.

File: pkgs/servers/plexpass/default.nix (hunk 1 above)
Comment (colemickens) [inline pkgs/servers/plexpass/default.nix]: When I copy the meta section from the source `plex` definition and make relevant modifications, I get this error:

```
error: cannot coerce a set to a string, at /nixpkgs/pkgs/servers/plexpass/default.nix:4:3
```

When I resolve it, I'll update this PR. If you have any hints, I'd appreciate it. Thanks.

**Comment by aszlig** [comment]
Just for reference, as we had this on IRC: It's maybe better to make this a flag for plex instead of using `overrideDerivation`.

**Comment by colemickens** [comment]
This has been updated to be a flag (`enablePlexPass`) on the existing `plex` package.

This should address your feedback @jagajaga since the meta section is now there. Please let me know if there is a more appropriate commit message then what I have here.

**Comment by colemickens** [comment]
Updated again to minimize duplication of `if`s based on @aszlig's suggestion and guidance.

**Comment by jagajaga** [comment]
Nice PR! :)

**Pull request merged**